mkdocs.yml

```diff
@@ -573,7 +573,6 @@ nav:
         - 'Release process and roadmap': 'community_resources/release_process.md'
         - 'Support and Maintenance FAQ': 'community_resources/support_maintenance_faq.md'
         - Contributing:
-            - 'Install Ibexa DXP on macOS or Windows': 'community_resources/installing-on-mac-os-and-windows.md'
```

Comment:
Are you removing the article from the TOC completely to discourage users from installing the product in these platforms?

Reply:
It is still findable via link/search, but yes, it is removed from the TOC to avoid indicating that installing on those systems is recommended.